Portal/vip-gatelab/src/main/java/fr/insalyon/creatis/vip/gatelab/client/GateLabModule.java

```diff
@@ -47,18 +47,24 @@
 public class GateLabModule extends Module {
 
     public GateLabModule() {
-      new ApplicationModule();
+      //new ApplicationModule();
     }
 
     @Override
     public void load() {
-
+        
         if (CoreModule.user.isSystemAdministrator()
                 || CoreModule.user.hasGroupAccess(GateLabConstants.GROUP_GATELAB)) {
-            for(String GateLabClass:ApplicationModule.reservedClasses){
+            for(String GateLabClass:ApplicationModule.reservedGateClasses){
             CoreModule.addApplicationsTileGrid(new GateLabTileGrid(GateLabClass));
             }
         }
+        if (CoreModule.user.isSystemAdministrator()
+                || CoreModule.user.hasGroupAccess(GateLabConstants.GROUP_GATELABTEST)) {
+            for(String GateLabClass:ApplicationModule.reservedTestClasses){
```

Comment:
This solution works for 2 GATE classes, but it is very ad-hoc. I would rather do the following:
for(String GateLabClass:ApplicationModule.reservedGateClasses){
  // Check which groups can access this class.
  // Check if user has access to one of these groups. If yes, addApplicationsTileGrid. 
}